index.bs

```diff
@@ -843,26 +843,31 @@ each other. A shared rendering engine may form an [=equivalence class=], for exa
 
 To <dfn>create an arbitrary brand</dfn>, the [=user agent=] MUST run these steps:
 
-    1. Let |arbitraryBrand| be a [=/string=] composed of [=ASCII alpha=]. |arbitraryBrand| MUST contain one or more
-         0x20 (SP) bytes and be no longer than twenty [=ASCII bytes=].
-    1. Let |arbitraryBrandList| be the result of <a lt="split on ASCII whitespace">splitting |arbitraryBrand| on ASCII
-         whitespace</a>.
+    1. Let |arbitraryBrand| be a [=/string=] composed of [=ASCII alpha=]. |arbitraryBrand| MUST
+        contain one or more 0x20 (SP) bytes and be no longer than twenty [=ASCII bytes=].
```

Comment:
what if arbitraryBrand had a space first? So then split would return `["","whatever"]`? I think you'd end up with a leading special character then. This might be fine though, given it says "composed of ASCII alpha", and I guess we could also leave open the possibility someone could very explicitly decide to start their string with a special character if they wanted to deal with it.

Reply:
That is a very good point. Let me update - let's make sure we don't start or end the string with 0x20.